openpifpaf/show/painters.py

```diff
@@ -479,3 +479,9 @@ def _draw_decoding_order(ax, decoding_order):
             ax.text(0.5 * (jsxyv[0] + jtxyv[0]), 0.5 * (jsxyv[1] +jtxyv[1]),
                     '{}: {} -> {}'.format(step_i, jsi, jti), fontsize=8,
                     color='white', bbox={'facecolor': 'black', 'alpha': 0.5, 'linewidth': 0})
+
+PAINTERS = {
```

Comment:
Don't define constants at the bottom of a file. Move this to `show/__init__.py`.

Reply:
Hmm, does that lead to a circular import?

Reply:
Yeah, that was the issue.

Reply:
In that case, move `PAINTERS` and `AnnotationPainter` to a new file `show/annotation_painter.py`. There you can keep `PAINTERS` at the top of the file.